notebooks/COS/Requirements.txt

```
@@ -0,0 +1,20 @@
+## Requirements
+python3.6
+numpy 
+matplotlib
+astropy
+astroquery
+glob
+sys
+os
+shutil
+datetime
+csv
+calcos
+costools
+scipy
+urllib
+tarfile
+specutils
+pandas
+IPython
```

```suggestion
ipython
```
the name you use here is the name the package goes by on `pypi`, not the import name.  For everything else it's the same, but IPython pre-dates the established conventions...

---

Resolved this too.